src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using SFA.DAS.LevyTransferMatching.Web.Controllers;
+using SFA.DAS.LevyTransferMatching.Web.Models.Pledges;
+
+namespace SFA.DAS.LevyTransferMatching.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class PledgesControllerTests
+    {
+        private PledgesController pledgesController;
+
+        [SetUp]
+        public void Setup()
+        {
+            pledgesController = new PledgesController();
+        }
+
+        [Test]
+        public void GET_Index_Returns_Expected_View_With_Expected_ViewModel()
+        {
+            // Arrange
+            string encodedAccountId = "abc";
```

Comment:
What about adding in the AutoFixture package and get it to create a random value for us?